openmetadata-ui/src/main/resources/ui/src/components/Domain/DomainPage.component.tsx

```diff
@@ -176,10 +177,6 @@ const DomainPage = () => {
     }
   }, [rootDomains, domainFqn]);
 
-  if (domainLoading) {
-    return <Loader />;
-  }
```

Comment:
I guess we need to show loader somewhere

Reply:
Yeah loader is there. We are showing it. 
<img width="302" alt="image" src="https://github.com/user-attachments/assets/e9842f36-ccb6-4ad4-8880-4fd728218a1a">